src/main/java/Comments.java

```diff
@@ -25,7 +25,7 @@
   }
 
   public void foo() {
-    Type t = new Type("firstParameter", // this is supposed to be single line, using a lot of text to show off long lines
+    Type t = new Type("firstParameter", // this is supposed to be single line, using a lot of text to show off long line
```

Comment:
spelling mistake?

Reply:
> spelling mistake?

Bad grammar for sure.  The issue with the existing example is that line is 121 characters long.  I was being lazy and not re-creating the whole sentence to be grammatically correct and 120 characters long.